test/integration/022_bigquery_test/execution-project-models/model.sql

```diff
@@ -0,0 +1,5 @@
+{{ config(materialized='incremental') }}
+select 1 as id
+{% if is_incremental() %}
+this is a syntax error!
+{% endif %}
```

Comment:
I imagine you copy-pasted this from a different test case. I think the query can just be `select 1 as id`, no need for the extra complexity.

Reply:
Indeed it's copy pasted I was wondering if the incremental check could be useful as I saw it elsewhere